coderd/workspaces_test.go

```diff
@@ -271,9 +271,38 @@ func TestWorkspaceByOwnerAndName(t *testing.T) {
 		coderdtest.AwaitTemplateVersionJob(t, client, version.ID)
 		template := coderdtest.CreateTemplate(t, client, user.OrganizationID, version.ID)
 		workspace := coderdtest.CreateWorkspace(t, client, user.OrganizationID, template.ID)
-		_, err := client.WorkspaceByOwnerAndName(context.Background(), codersdk.Me, workspace.Name)
+		_, err := client.WorkspaceByOwnerAndName(context.Background(), codersdk.Me, workspace.Name, codersdk.WorkspaceByOwnerAndNameParams{})
 		require.NoError(t, err)
 	})
+	t.Run("deletedGetWorkspaceByOwnerAndName", func(t *testing.T) {
```

Comment:
Suggestion: it's nice to keep the test names short when you're typing out that big `go test ...` command :-)

```suggestion
	t.Run("Deleted", func(t *testing.T) {
```